packages/ui/react-ui/src/useEnvironment.ts

```diff
@@ -0,0 +1,10 @@
+import { useEffect, useState } from "react";
+
+export default function useEnvironment() {
+    const [isSSR, setIsSSR] = useState(true);
```

Comment:
is SSR a common enough term that it can be easily interpreted or should we expand it out into `isServerSideRendering`?